host/crypto/bcrypt/bcrypt.h

```diff
@@ -6,10 +6,12 @@
  * are defined in both Windows.h and ntstatus.h. The WIN32_NO_STATUS will
  * prevent these redefinitions.
  */
-#include <ntstatus.h>
+#define NOCRYPT
```

Comment:
Can you update the comment to reflect why NOCRYPT is being defined?

Reply:
Added a comment for this.